ui/js/directives/treeherder/clonejobs.js

```diff
@@ -900,7 +898,10 @@ treeherder.directive('thCloneJobs', [
         };
 
         var generateJobElements = function(resultsetAggregateId, resultset) {
-
+            ReactDOM.render(
+              React.createElement(window.Hello, {name: "Let's ReactJS!"}),
+              document.getElementsByClassName("job-list")[0]
```

Comment:
Are you sure .job-list is already there? I would rather try to add the new elements to 
```js
document.getElementById(resultsetAggregateId)
```